std/format.d

```diff
@@ -3040,19 +3040,47 @@ if (isInputRange!T)
             static if (is(StringTypeOf!T))
             {
                 auto s = val[0 .. f.precision < $ ? f.precision : $];
+
+                size_t width;
+                if (f.width > 0)
+                {
+                    // dstring & string that are fully made of ASCII characters
+                    // can be aligned w/o graphemeStride
```

Comment:
Why can't wstring containing only ASCII be aligned w/o graphemeStride?

Reply:
fixed